lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -73,6 +73,7 @@
 
     # spoc gradebook
     url(r'^gradebook$', gradebook_api.spoc_gradebook, name='spoc_gradebook'),
+    url(r'^mastersGradebook$', masters_gradebook_api.spoc_masters_gradebook, name='spoc_masters_gradebook'),
```

Comment:
The typical convention in python is to not use camel-casing.
```suggestion
    url(r'^masters_gradebook$', masters_gradebook_api.spoc_masters_gradebook, name='spoc_masters_gradebook'),
```